lib/cookies.js

```diff
@@ -9,11 +9,10 @@ module.exports = (options) => {
     if (req.cookies && Object.keys(req.cookies).length) {
       next();
     } else if (req.query[paramName] !== undefined) {
-      var err = new Error('Cookies required');
```

Comment:
Arguably this should be in another PR of course.
But it's so minor does anyone really care?